frontend/public/components/dashboard/dashboard-card/card.scss

```diff
@@ -2,8 +2,8 @@
   margin: 0;
 }
 
-.co-dashboard-card__body {
-  margin-top: 1.5em;
+.pf-c-card__body.co-dashboard-card__body {
```

Comment:
I wasnt able to override the default padding by passing `co-dashboard-card__body` because `pf-c-card__body` style took precedence over mine. It seems like Im missing some configuration somewhere, but couldnt figure it out

Reply:
It seems like this will be a common use case that PatternFly should support.  I recommend opening an upstream issue.

Rather than overriding PatternFly here (which can lead to a maintenance issue for us since it changes the default behavior), I recommend using padding and negative margins that match the `.co-dashboard-card__body` padding values on `.co-utilization-card__item` (see https://getbootstrap.com/docs/3.3/css/#grid for an example).  This way, the change is an explicit opt-in that we control, and we are not changing the behavior of the upstream, thus requiring an addition to every default `.co-dashboard-card__body`.

Reply:
Edited above comment to change `.co-inventory-card__item` to `.co-*-card__item` as these classes are card-specific.  Probably want a generic class here instead to control add negative margins and padding.

Reply:
@mcoker @matthewcarleton is there a recommended approach here?

Reply:
+1 to [opening an issue in PF](https://github.com/patternfly/patternfly-next/issues/new) for this support. As a temporary workaround, we do have a new experimental divider component you could use and define in between card sections. There is no padding on the card itself, so the divider would span the width of the card as long as it's not in one of the header, body, or footer sections. I've created a codepen to show one way of using it in the card to create section dividers - https://codepen.io/mcoker/pen/WNewzGg

Reply:
thank you all for suggestions, I went with negative margin approach as the divider does not go well with how we structure our components in dashboard. Issue reported https://github.com/patternfly/patternfly-next/issues/2153